doc/conf.py

```diff
@@ -99,12 +99,17 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'nature'
+html_theme = 'python_docs_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#html_theme_options = {}
+html_theme_options = {
+    'collapsiblesidebar': True,
+    'issues_url': 'https://github.com/pycqa/pylint/issues/new',
+    'root_name': 'PyCQA',
+    'root_url': 'https://pylint.pycqa.org',
```

Comment:
Not sure if this should be linked to http://meta.pycqa.org instead of https://pylint.pycqa.org?
This is the link that says `PyCQA` in the top left.

Reply:
We could probably use the meta one!

Reply:
Ok, changed.